R/metadata.R

```diff
@@ -14,11 +14,16 @@ cache <- rlang::env(
 #' @return A character scalar consisting of the URL
 #' @examples
 #' get_metadata(remote_url = DATABASE_URL)
-DATABASE_URL <- single_line_str(
-    "https://object-store.rc.nectar.org.au/v1/
-    AUTH_06d6e008e3e642da99d806ba3ea629c5/metadata/metadata.0.2.3.parquet"
-)
 
+
+DATABASE_URL <- function(databases = c("metadata.0.2.3.parquet", "fibrosis.0.2.3.parquet")) {
```

Comment:
@myushen it is just a style comment. But I think we should keep consistent with the function name styling

i.e. consistent with this `get_default_cache_dir`